libs/sdk-ui-ext/src/internal/interfaces/Visualization.ts

```diff
@@ -58,6 +58,7 @@ export interface IVisConstruct {
 export interface ICustomProps {
     drillableItems?: ExplicitDrill[];
     totalsEditAllowed?: boolean;
+    lastSavedVisClassUrl?: string;
```

Comment:
What this url refers to? What does it mean "saved"?

Reply:
It is the url in visualization class of the current opening insight. It will be used for detecting the case users open an insight which is not headline and then switch to headline. 
The "lastSaved" is connected to this name  "REPORT_LAST_SAVED_OBJECT_DATA" in https://github.com/gooddata/gdc-analytical-designer/blob/develop/app/constants/state_paths.ts#L202

Reply:
Thank you for the explanation.